frontend/README.md

```diff
@@ -1,8 +1,30 @@
 ## Available Scripts
 
-In the project directory, you can run:
-### `npm install`
-### `yarn run dev-server`
+This application can be run locally with webpack-dev-server or via a docker container.  Both methods below will result in a web site running in development mode available at http://localhost:8080
 
-Runs the app in the development mode.<br>
-Open [http://localhost:8080](http://localhost:8080) to view it in the browser.
+#### To run locally via webpack-dev-server:
+```
+$> npm install
+$> npm run dev-server
+```
+Visit http://localhost:8080 in your browser.
```

Comment:
This now opens up on port 3000 for me. Can you confirm the port?

Reply:
Docker maps the ports so you should still be hitting port 8080 for nginx to proxy to node

Reply:
My point was actually regarding the instructions w/o docker, but I think it's probably a local issue that's causing the port to go to 3000.